openapi/paths/capital-projects.yaml

```
@@ -0,0 +1,23 @@
+get:
+  summary: Find paginated capital projects.
+  operationId: findCapitalProjects
+  tags:
+    - Capital Projects
+  parameters:
+    - $ref: ../components/parameters/limitParam.yaml
+    - $ref: ../components/parameters/offsetParam.yaml
+  responses:
+    '200':
+      description: >-
+        An object containing pagination metadata and an array of capital
+        projects
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/CapitalProjectPage.yaml
+    '400':
+      $ref: ../components/responses/BadRequest.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
```

The capital projects endpoint does not currently have a case where it 404s. (And I think the upcoming query parameter validations will also only cause 400)

```suggestion
    '404':
      $ref: ../components/responses/NotFound.yaml
```